backend/src/main/java/com/itlize/joolemarketplace/service/impl/ProjectProductServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.itlize.joolemarketplace.service.impl;
+
+import com.itlize.joolemarketplace.model.Product;
+import com.itlize.joolemarketplace.model.Project;
+import com.itlize.joolemarketplace.model.ProjectProduct;
+import com.itlize.joolemarketplace.repository.ProjectProductRepository;
+import com.itlize.joolemarketplace.service.ProjectProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Optional;
+
+public class ProjectProductServiceImpl implements ProjectProductService {
+    @Autowired
+    private ProjectProductRepository projectProductRepository;
```

Comment:
Do you need projectRepository and productRepository here?
Just like the one in Korera ColumnsServiceImpl. In create method, it uses the projectRepository

Reply:
Good point. I could be wrong, but I think it makes more sense for `ProjectServiceImpl` to handle operations like `addProductToProject` so we can leave those out here.

Reply:
@chelsea-chenxi1999 for viz